examples/landmarks/at.html

```diff
@@ -281,4 +281,4 @@ <h2 id="id3">Related W3C Documents</h2>
     </script>
     <script type="text/javascript" src="js/SkipTo.min.js"></script>      
   </body>
-</html>
```

Comment:
Can you please use an editor that supports .editorconfig so we get consistent formatting. Files should end in a newline